src/styles/spacing/index.md.njk

```diff
@@ -193,6 +193,18 @@ For example:
 - `govuk-!-padding-right-5` will apply 15px of padding to the right side of the element on small screens, increasing to 25px on large screens
 - `govuk-!-margin-0` will remove all margins at all screen sizes
 
+### Static spacing override classes
+
+The static spacing override classes start with `govuk-!-static` followed by either `margin-` or `padding-`, and then the spacing unit you want to apply.
+
+To apply spacing in a single direction, include `left-`, `right-`, `top-`, or `bottom-` between `margin-` or `padding-` and the spacing unit.
```

Comment:
There's a lot of repetition in this section – the only difference between these two paragraphs and the corresponding paragraphs for responsive spacing is the extra 'static' in the class name.

One alternative would be to describe these in comparison to the responsive spacing scale, like:

```suggestion
The static spacing override classes work the same way as the responsive spacing override classes, but start with `govuk-!-static`.
```

Possibly less confusing to see what the difference is, but makes this section less standalone?